.github/workflows/ci.yml

```diff
@@ -17,4 +17,10 @@ jobs:
       with:
         fetch-depth: 0
     - run: sudo rm -rf /usr/local/bin/kustomize
+    - run: export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+    - run: export OS=$(uname | awk '{print tolower($0)}')
+    - run: export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.15.0
+    - run: curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+    - run: chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
     - run: make test
+    - run: make bundle && git diff --exit-code
```

Comment:
For just the CRD, I would rely on 'make manifests' and 'make generate' and check their exit codes. Then you wouldn't need the other code either. It might also make sense to separate these out into other checks? So basically a 'check manifests' and 'check generated code', so that it's explicit as to what failed.